.github/workflows/android-arm64.yaml

```diff
@@ -2,7 +2,7 @@ name: android-arm64
 
 on:
   pull_request:
-    types: [opened, reopened, edited, synchronize, labeled]
+    types: [ready_for_review, synchronize, labeled]
```

Comment:
what about line #13:

if: github.event_name != 'label' || github.event.label.name == 'runtest'

I think we want to remove that.

Reply:
Done